### PR TITLE
[10.0] MIG l10n_it_rea

### PR DESCRIPTION
**Pull request by andrea4ever**

### CODE REVIEW

File: l10n_it_rea/README.rst [hunk 1]
```diff
@@ -33,6 +33,7 @@ Contributors
 * Alessio Gerace <alessio.gerace@agilebg.com>
 * Nicola Malcontenti <nicola.malcontenti@agilebg.com>
 * Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Andrea Gallina <a.gallina@apuliasoftware.it>
```
Comment (tafaRU) [inline l10n_it_rea/README.rst]: Could you please take the occasion to apply to the README file the following changes?

- add `:target: http://www.gnu.org/licenses/agpl-3.0-standalone.html `to the badge license 
- put ============ above REA Register
- remove content of Usage section to the previous section
- add Try me on Runbot lunk to Usage section
-  use https://odoo-community.org

See https://raw.githubusercontent.com/andrea4ever/l10n-italy/5be3170070d5c8b54b945b51f673dcd8d7b2c468/l10n_it_rea/README.rst as refernce

File: l10n_it_rea/__manifest__.py [hunk 2]
```diff
@@ -1,22 +1,8 @@
...
-#
-##############################################################################
+# -*- encoding: utf-8 -*-
+# Copyright 2014 Associazione Odoo Italia (<http://www.odoo-italia.org>)
+# Copyright 2015 Alessio Gerace <alessio.gerace@agilebg.com>
+# Copyright 2016 Andrea Gallina (Apulia Software)
+
 {
     'name': 'REA Register',
     'version': '9.0.0.1.0',
```
Comment (tafaRU) [inline l10n_it_rea/__manifest__.py]: Fix the version number

File: l10n_it_rea/views/partner_view.xml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```
Comment (tafaRU) [inline l10n_it_rea/views/partner_view.xml]: Please use odoo tag

File: l10n_it_rea/README.rst (hunk 1 above)
Comment (andrea4ever) [inline l10n_it_rea/README.rst]: @tafaRU done!

**Pull request merged**